src/index.html

```diff
@@ -4,9 +4,433 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,100&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__img"
+          src="./images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active nav__link--is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#SAMSUNG">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#SMARTPHONES">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#LAPTOPS & COMPUTERS"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#GADGETS">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#TABLETS">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#PHOTO">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#VIDEO">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card-holder">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product-image">
+
+        <div class="card__product-info">
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <h2 class="card__product-code">
+            Product code: 195434
+          </h2>
+
+          <div class="stars-Wrapper">
+            <div class="stars-Wrapper__row stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+
+            <span class="stars-Wrapper__card-reviews">
+              Reviews: 5
+            </span>
+          </div>
```

Comment:
```suggestion
          <div class="card__rating">
            <div class="stars stars--4">
              <div class="stars__star star"></div>
              <div class="stars__star star"></div>
              <div class="stars__star star"></div>
              <div class="stars__star star"></div>
              <div class="stars__star star"></div>
            </div>

            <span class="card__rating-reviews">
              Reviews: 5
            </span>
          </div>
```

Use proper BEM structure